src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -147,7 +147,11 @@ func main() {
 		logger.Infof("Finished all")
 	}()
 
-	go assistedController.WaitAndUpdateNodesStatus(mainContext, &wg)
+	removeUninitializedTaint := false
+	if cluster.Platform != nil && *cluster.Platform.Type == models.PlatformTypeVsphere {
```

Comment:
I think we should add a condition about OCP version >= 4.13

Reply:
This taint is used by external cloud controller only, so it will not appear in 4.12 or less